### DISCUSSION TITLE
Unexpected type supplied to modifier

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#5939] [Troubleshooting]
Hi

Since updating to 3.3 I've noticed this error

```
[{
	"resource": "/Users/xxxxxxxx/Repositories/xxxxxxxx/resources/views/structured-data/_local-business.antlers.html",
	"owner": "_generated_diagnostic_collection_name_#1",
	"severity": 4,
	"message": "[ANTLR_508] Unexpected type supplied to modifier . Expected string got array.",
	"source": "antlers",
	"startLineNumber": 42,
	"startColumn": 55,
	"endLineNumber": 42,
	"endColumn": 60
}]
```

It's referring to this

```
"latitude":"{{ company:long_lat | explode:, | first }}",
```

`first` is highlighted.

What's the correct way to fix this?

It's worth mentioning, it does work as it is. 

I'd just like to know if there's a way to stop the error from showing up.

**Comment by JohnathonKoster** [comment]
Hi there!

I've just published version `2.0.14` of the VS Code extension that improves the error reporting for the `first` and `last` modifiers. Currently there is no way to entirely disable error reporting (but feel free to create an issue/FR over here: https://github.com/Stillat/vscode-antlers-language-server)

**Reply by stephenmeehanuk** [reply]
Will do, thanks.

**Comment by edalzell** [comment]
> I'd just like to know if there's a way to stop the error from showing up.

My guess is the modifier needs fixing, submit an issue please.

**Reply by stephenmeehanuk** [reply]
Updating the Antlers plugin to 2.0.14 fixed the issue.
